app/models/application.rb

```diff
@@ -137,7 +137,7 @@ def inside_uk_catchment?
   end
 
   def inside_catchment?
-    %w[england].include?(teacher_catchment)
+    %w[england].include?(teacher_catchment) || (cohort.start_year < 2024 && !!school&.urn&.starts_with?("1"))
```

Comment:
I think considering this is a 'fix' for declarations against the legacy applications I'd rather this changed the check in Declaration rather than changing the definition of inside_catchment? on Application